src/test/unittest/devdax.py

```diff
@@ -48,6 +61,12 @@ def try_assign(self, path):
         if self._req_alignment and p_align != self._req_alignment:
             return False
 
+        is_deep_flush = self.check_deep_flush_file(path)
+
+        if (self._req_deep_flush is True and is_deep_flush is False) or \
+           (self._req_deep_flush is False and is_deep_flush is True):
```

Comment:
I would have expected that the `deep_flush=false` means that deep flush isn't required to be supported, not that it can't be.
I think this should be somehow clarified - maybe by changing the variable names or adding a comment somewhere.